packages/web-components/src/components/va-file-input/va-file-input.scss

```diff
@@ -4,15 +4,26 @@
 @use 'usa-label/src/styles/usa-label';
 @use 'usa-hint/src/styles/usa-hint';
 @use 'uswds-helpers/src/styles/usa-sr-only';
+
 @import '../../mixins/uswds-error-border.scss';
 @import '../../global/formation_overrides';
+@import '../../mixins/focusable.css';
 
-/* V3/USWDS Styles */
 [hidden] {
   display: none;
 }
 
-/* Original Styles */
+.usa-file-input__input {
+  box-sizing: border-box; // Formation override.
+  font-family: var(--font-source-sans);
```

Comment:
The font was loading as Arial so I'm making it Source Sans here. We believe that it might be a bug on the USWDS site which is loading it as Public Sans. 

Original convo here:

https://github.com/department-of-veterans-affairs/component-library/pull/1017#issuecomment-1906220668